packages/iot-app-kit-visualizations/src/components/charts/sc-tooltip/sc-tooltip.tsx

```diff
@@ -29,7 +29,7 @@ export class ScTooltip {
   @Prop() baseChartRef: HTMLElement;
 
   @Prop() size!: SizeConfig;
-  @Prop() dataContainer!: HTMLElement;
+  @Prop() dataContainer: HTMLElement | null;
```

Comment:
Curious how the `dataContainer` can be unavailable during `componentDidLoad`?

Reply:
It might be a render sequencing issue? I think the tooltip must be rendered before  the DataContainer component

Reply:
The method to get data container is just a querySelector, so it's not ensured to be in the document.